svghmi/widget_jsontable.ysl2

```diff
@@ -189,8 +189,13 @@ template "svg:text", mode="json_table_elt_render" {
     }
 }
 
+template "svg:image", mode="json_table_elt_render" {
+    param "expressions";
+    const "value_expr", "$expressions/expression[1]/@content";
+    |         id("«@id»").setAttribute('xlink:href', String(«$value_expr»));
```

Comment:
"xlink:href" is deprecated. This should be accessible as an object member with ".href"